### PR TITLE
New icons (updated the OG images, tags)

### PR DESCRIPTION
**Pull request by dburka**
Added Covid icons, a bunch of Body icons, and a few Symbols as well.

### CODE REVIEW

File: public/icons/meta-data.json [hunk 1]
```diff
@@ -6940,5 +6940,392 @@
...
+   "virus"
+  ],
+  "title": "Quarantine Home"
+ },
+ {
+  "id": "calendar-quarantine",
+  "category": "symbols",
+  "path": "symbols/calendar-quarantine",
+  "tags": [
+   "calendar",
```
Comment (nburka) [inline public/icons/meta-data.json:6962]: lower case tags should be title case

File: public/icons/meta-data.json [hunk 2]
```diff
@@ -6940,5 +6940,392 @@
...
+   "Space"
+  ],
+  "title": "Social Distancing ALT"
+ },
+ {
+  "id": "intestinal-pain",
+  "category": "symptoms",
+  "path": "symptoms/intestinal-pain",
+  "tags": [
+   "ache",
```
Comment (nburka) [inline public/icons/meta-data.json:6988]: more lower case tags

File: public/icons/meta-data.json [hunk 3]
```diff
@@ -6940,5 +6940,392 @@
...
+   "symptom"
+  ],
+  "title": "Intestinal Pain"
+ },
+ {
+  "id": "virus-shield",
+  "category": "symbols",
+  "path": "symbols/virus-shield",
+  "tags": [
+   "covid",
```
Comment (nburka) [inline public/icons/meta-data.json:7002]: lower case

File: public/icons/meta-data.json [hunk 4]
```diff
@@ -6940,5 +6940,392 @@
...
+   "virus"
+  ],
+  "title": "Shield Virus"
+ },
+ {
+  "id": "syringe-vaccine",
+  "category": "devices",
+  "path": "devices/syringe-vaccine",
+  "tags": [
+   "covid",
```
Comment (nburka) [inline public/icons/meta-data.json:7014]: lower case

**Review by nburka — CHANGES_REQUESTED** [review]

**Comment by dburka** [comment]
Fixed the lowercase tags. Will merge. Thanks @nburka

**Pull request merged**